checkoutmanager/runner.py

```diff
@@ -82,6 +82,25 @@ def execute_action(dirinfo, custom_actions, action):
         return e
 
 
+def run_one(action, directory=None, url=None, conf=None):
+    custom_actions = get_custom_actions()
+    if not conf:
+        conf = config.Config(os.path.expanduser(CONFIGFILE_NAME))
+    from .dirinfo import DirInfo
```

Comment:
I assume the import is here because of recursive imports?

Note: I prefer full imports; all the other imports are :-)